lib/plivo/resources/call_feedback.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+require 'faraday'
+require 'faraday_middleware'
+
+module Plivo
+  module Resources
+    include Plivo::Utils
+    class CallFeedback < Base::Resource
+      def initialize(client, options = nil)
+        @_name = 'Call'
+        super
+      end
+    end
+
+    class CallFeedbackInterface < Base::ResourceInterface
+      FEEDBACK_API_PATH = "v1/Call/%s/Feedback/"
+      def initialize(client, resource_list_json = nil)
+        @_name = 'Call'
+        @_resource_type = CallFeedback
+        super
+      end
+
+      def create(call_uuid, rating, issues = [], notes = "")
+        
+        valid_param?(:call_uuid, call_uuid, String, true)
```

Comment:
we can check if call_uuid and rating are provided, are valid and then proceed ahead.